tests/devices/test_default_qutrit.py

```diff
@@ -1391,3 +1396,527 @@ def test_density_matrix_all_wires(self, qutrit_device_2_wires, wires, expected):
         qutrit_device_2_wires.apply(ops)
 
         assert np.allclose(qutrit_device_2_wires.density_matrix(wires), expected)
+
+
+# pylint:disable=no-member
+
+
+# JAX integration tests
+@pytest.fixture(scope="class")
+def jax_import_init(request):
+    request.cls.jax = pytest.importorskip("jax")
+    request.cls.jnp = pytest.importorskip("jax.numpy")
+    request.cls.config = pytest.importorskip("jax.config").config
+
+
+@pytest.mark.jax
+@pytest.mark.usefixtures("jax_import_init")
+class JaxIntegrationTest:
+    pass
+
+
+class TestQNodeIntegrationJax(JaxIntegrationTest):
```

Comment:
For each of the jax tests, it would be worth running them with and without jit.

```
if use_jit:
    circuit = jax.jit(circuit)
```

Note that the device state might not be right in that case because the device state getting updated is a side effect.

Reply:
Added

Reply:
I am finding an issue with broadcasting when using jax .jit on a circuit, it is just repeating the first result x times. I found the same issue with default.mixed class it is fixed in default.qutrit, but I think that is due to the new device API. Should I try to fix this or should I leave it? Broadcasting works without jitting.

Reply:
This problem seems to be attached to #2762. @albi3ro is it ok to leave this for now? should I make a TODO comment on the test for when #2762 is fixed, should I add a comment to the issue that it affects qutrits?

Reply:
There is lots wrong with the old device API. If you see this in `default.mixed` and `default.qubit.legacy` but not in `default.qubit`, I think it's a good idea to address it in a separate PR. @albi3ro do you agree?

Reply:
In my opinion you don't need to fix it. That bug is unrelated to this PR, and as far as I can tell the only device affected is `default.mixed`. You're welcome to attempt to fix it if you'd like though 😄 ! However, that should be in a separate PR.